app/serializers/api/variant_serializer.rb

```diff
@@ -38,4 +39,12 @@ def thumb_url
       "/noimage/mini.png"
     end
   end
+
+  def unit_price_price
+    (rand * 10).round(2)
+  end
+
+  def unit_price_unit
+    rand > 0.5 ? "item" : "kg"
+  end
```

Comment:
I don't mean to change it because this is just temporal but I was curious and went check Ruby docs and the `Random` class has a way to do this using `#rand(max)` :point_right: https://ruby-doc.org/core-2.4.0/Random.html#method-c-rand :ok_hand: